runtime/compiler/control/J9Options.cpp

```diff
@@ -1106,7 +1106,7 @@ J9::Options::fePreProcess(void * base)
          case j9gc_modron_wrtbar_cardmark:               wrtbarMode = TR_WrtbarCardMark; break;
          case j9gc_modron_wrtbar_cardmark_and_oldcheck:  wrtbarMode = TR_WrtbarCardMarkAndOldCheck; break;
          case j9gc_modron_wrtbar_cardmark_incremental:   wrtbarMode = TR_WrtbarCardMarkIncremental; break;
-         case j9gc_modron_wrtbar_realtime:               wrtbarMode = TR_WrtbarRealTime; break;
+         case j9gc_modron_wrtbar_satb:                   wrtbarMode = TR_WrtbarRealTime; break;
```

Comment:
we should have j9gc_modron_wrtbar_satb_and_oldcheck defined and checked at various spots where j9gc_modron_wrtbar_satb is checked

Reply:
fixed